configure.ac

```diff
@@ -111,6 +111,18 @@ AX_TLS(,:)
 test "$ac_cv_tls" != "none" && lg_tls=yes
 
 AX_PTHREAD
+if [ test -n "$ax_pthread_ok" ]; then
+	CC="${PTHREAD_CC:-CC}"
+	#CCX="${PTHREAD_CC:-CCX}"
+
+	dnl Currently, libtool uses --nostdlib when linking shared libraries,
+	dnl which causes the -pthread flag to be ignored. Try to work around that
+	dnl by explicitly specifying the pthread library unless it is already set.
+	if [ test -z "$PTHREAD_LIBS" ]; then
+		PTHREAD_LIBS=-lpthread
+		LIBS="$PTHREAD_LIBS $LIBS"
```

Comment:
I'll merge this, but then remove line 123 because, I believe, only liblinkgrammar needs to be linked this way. This would match an earlier change you'd make